packages/k2-alpine/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@avalabs/k2-alpine",
+  "license": "Limited Ecosystem License",
+  "version": "1.0.0",
+  "main": "expo/AppEntry.js",
+  "files": [
+    "./src/index.ts"
+  ],
+  "scripts": {
+    "start": "expo start",
+    "android": "expo start --android",
+    "ios": "expo start --ios",
+    "lint": "eslint .",
+    "storybook-generate": "sb-rn-get-stories",
+    "tsc": "tsc -p ."
+  },
+  "dependencies": {
+    "dripsy": "^4.3.5",
+    "expo": "~51.0.28",
+    "expo-status-bar": "~1.12.1",
+    "react": "18.2.0",
+    "react-native": "0.74.5",
+    "react-native-reanimated": "^3.15.0"
```

Comment:
you will also need to specify some dependencies such as react, react-native, react-native-reanimated,...in peerDependencies. otherwise, core mobile will not be able to resolve the dependencies correctly due to conflicts. these conflicts should show up right away the moment core mobile tries to consume k2-alpine.

Reply:
on a similar note, you may want to keep these dependencies as the same versions as what are in core mobile to prevent incompabilitiy. the ones that core mobile will use will be the ones that are from core mobile's node_modules.